### ISSUE TITLE
Live stream to video overlay

### ISSUE THREAD

**Issue by moderniseobservatory** [ronikar/Leaflet.DistortableVideo#2]
Is there a way to stream a live video this overlay? I am working on a surveillance / search and rescue project that utilizes drones and maps and having live footage would be very important.

I have thought about encoding on the go but that doesn't seem like it will work... I think i need to use MPEG-DASH but I'm unsure how/ if its possible.

Thanks for any suggestions.

**Comment by ronikar** [comment]
Try first to stream a live video with HTML video element (`<video>`). Then, pass this video element as parameter to `L.distortableVideoOverlay`.

**Comment by moderniseobservatory** [comment]
Pass it as a parameter in place of the source url? 
How/ where can i reference a HTML element in this function?
`  L.distortableVideoOverlay(  PARAMS  ).addTo(this.map);`

**Comment by ronikar** [comment]
yes
`L.distortableVideo(videoElement, corners, options).addTo(map)`

**Comment by moderniseobservatory** [comment]
Yep, finally got it to work. What confused me was my own work /.-
As I'm working in Angular5 with typescript, I needed to write a typings file to properly import your plugin. In my declaration I specified that the URL could only be a string
`export function distortableVideoOverlay(url: string, corners: Corners, options: object): Control.DistortableVideoOverlay;`

The fix was simply adding the HTMLVideoElement type to the URL declaration.
`export function distortableVideoOverlay(url: string | HTMLVideoElement, corners: Corners, options: object): Control.DistortableVideoOverlay;`

Thanks for your help!
